### PR TITLE
Add: home page

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
- Add image fix 
- Added carousel

### CODE REVIEW

**Review by Shah-Arsalan — APPROVED** [review]
Good to go

File: index.html [hunk 1]
```diff
@@ -90,12 +91,148 @@ <h1 class="tui__nav--logo tui__h-3">
...
+            </div>
+          </div>
+        </section>
+      </div>
       <h1 class="tui__h-1">Log<sub>2</sub></h1>
       <p class="tui__text-lg">
-        Log<sub>2</sub> is a store dedicated to selling books that fall around
-        computers and related stuff. Check out detailed list of products
-        <a href="/products">(here)</a>.
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (avindhawale) [inline index.html:146]: Any specific reason to use &nbsp; for spacing, you can use text-indent instead.

Comment (SJTGSHIVAM) [reply index.html:146]: With text-indent it is difficult to define the exact spacing for the next words to start. Especially with the responsive nature of fonts. You see the space char typically synchronizes with the rest of the font sizes and always starts at the right place where I am needing it to be.
There is one more way to achieve the same behavior that is if I fix these to text pieces together with the usage of relative and absolute positioning but again those pieces sound good but act differently. Moreover, I find it to be overkill to apply that. I even thought of implementing with grids but turned out that was also a lot hacky.
One more possible way was with the usage of ex but that too was too tricky in some sense (so I overlooked that one).
Apart from being simple I also found it to be a lot intuitive to use it this way. I believe I definitely need to read more literature on the same but as per the things I went through previously. It felt right to employ the spacing char in the same place.

Comment (avindhawale) [reply index.html:146]: Didn't get where/how exactly you want to align this text. 
Try to get fix in next commit.

**Review by avindhawale — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (SJTGSHIVAM) [reply index.html:146]: Let's have a discussion today evening. We can surely work on this.

**Review by palash23jain — APPROVED** [review]
good to go

File: index.html (hunk 1 above)
Comment (avindhawale) [reply index.html:146]: Sure. Would like to work on it.

**Pull request merged**